sdk/logz/Microsoft.Azure.Management.Logz/src/Microsoft.Azure.Management.Logz.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Logz</PackageId>
+    <Description>Microsoft Azure Management Logz library</Description>
+    <Version>1.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.Logz</AssemblyName>
+    <PackageTags>management;Logz;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is a public release of the Azure Logz SDK.
```

Comment:
This will become release notes for the nuget package, use "First release of the Azure Logz SDK for .Net"

Reply:
changed the text as recommended